ports/saucer/portfile.cmake

```
@@ -11,10 +11,15 @@ vcpkg_from_github(
         fix-source-generation.patch
 )
 
+set(BACKEND_OPTION)
 if(VCPKG_TARGET_IS_WINDOWS)
-    set(BACKEND_OPTION "-Dsaucer_backend=WebView2")
+    if("webview2" IN_LIST FEATURES)
+        set(BACKEND_OPTION "-Dsaucer_backend=WebView2")
+    endif()
```

```suggestion
    if("webview2" IN_LIST FEATURES)
        set(BACKEND_OPTION "-Dsaucer_backend=WebView2")
    endif()
```

This does nothing. The default backend on windows is WebView2 https://github.com/saucer/saucer/blob/3073fc169f2d2736de115f20c90451068b91f76b/CMakeLists.txt#L58

---

Yes, webview2 is installed by default on windows platform, I will remove it.